server/routes/api.js

```diff
@@ -48,4 +48,74 @@ router.get('/users', (req, res) => {
     });
 });
 
+router.get('/Students', (req, res) => {
+  connection((db) => {
+  db.collection('Students')
+    .find()
+    .toArray()
+    .then((users) => {
```

Comment:
This should be students here not users.  Technically it works because you are just naming a variable here, but it probably should be changed to avoid confusion later one.

There are two for each of the routes that need to be changes to the respective route